proposals/shared-everything-threads/Overview.md

```diff
@@ -143,6 +150,15 @@ The syntax of `globaltype` is extended:
 globaltype ::= share mut valtype
 ```
 
+Similarly, the text format is extended:
+
+```
+globaltype ::= t:valuetype => unshared const t
+             | '(' 'mut' t:valuetype ')' => unshared var t
+             | '(' 'shared' t:valuetype ')' => shared const t
+             | '(' 'shared '(' 'mut' t:valuetype ')' ')' => shared var t
```

Comment:
I currently have `(shared mut t:valuetype)` implemented here. I thought `shared` and `mut` both modified the global; this would imply that `mut` modifies the valuetype?

Reply:
I figured that if `mut` introduces parens and `shared` introduces parens, then using both would introduce both parens. This is similar to the situation with `(type (sub (shared (struct ...))))`.